packages/dds/tree/package.json

```diff
@@ -11,18 +11,42 @@
 	"license": "MIT",
 	"author": "Microsoft and contributors",
 	"sideEffects": false,
+	"type": "module",
```

Comment:
Isn't this a bit dangerous?

Reply:
I don't know that this is dangerous. Why might it be dangerous?

Reply:
(I haven't requested the tsc-multi feature that I think it needs. Without tsc seeing "type": "module" it doesn't go into full ESM mode. This is a must for now to ensure proper ESM transpilation.)